### ISSUE TITLE
Overwatch button missing

### ISSUE THREAD

**Issue by ShinigamiGamingInc** [ValveSoftware/Dota2-Gameplay#15446]
### Description

After all the failed overwatch replays, now the overwatch button in missing. Normaly restarting the game fixed it, but this time nothing is happening. It's just missing.

**Comment by SpiritWinD86** [comment]
Probably it was removed until they'll fix it.  Overwatch replays are broken rn anyway.

**Comment by ShinigamiGamingInc** [comment]
It was replays in general, but it seams they fixed the normal replays. Don't think overwatch would be that much different.

**Comment by SpiritWinD86** [comment]
> It was replays in general, but it seams they fixed the normal replays. Don't think overwatch would be that much different.

Yes. Well, this is not the first time this has happened, previously this also often happen before major updates, and given that the next update with the arcana for Skywrath and Venge will be released soon, then perhaps this can be the case. Just wait for a few days. Pretty sure they'll fix this soon.

**Comment by ShinigamiGamingInc** [comment]
But I neeeeed it.
Well I need it for my overwatch series. Let's hope for the best.

**Comment by SpiritWinD86** [comment]
Well, looks like they don't even trying to fix this or another small broken things before big update (which is delayed again) or until someone creates a thread on /r/DotA2/ (or bug ticket here on github) with a lot of upvotes.
